### PR TITLE
create Rachitt-shah's namecard

### PR DESCRIPTION
**Pull request by godslayer201**

### CODE REVIEW

**Comment by godslayer201** [comment]
Create name card

File: index.html [hunk 1]
```diff
@@ -12970,35 +12970,44 @@ <h2>A project for first-time contributions</h2>
 
             <!-- ________ Contributor's card START ________  -->
             <div class="card">
-                <p class="name">Name</p>
+                <p class="name">Rachitt Shah</p>
```
Comment (sanilborkar) [inline index.html:12973]: It looks like you've modified the contributor's card template. You need to copy this template and paste it above the line
```
<!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
```
You can refer to the other cards in this file for reference or reach out to us if you need any help.

**Comment by godslayer201** [comment]
@sanilborkar,my apologies, I'll rectify the mistake and do it again. Also,thank you for the opportunity!

**Comment by Syknapse** [comment]
I will now close this PR and hope you open a new one with the requested changes. Please don't replace the template contributor card, just copy it and add your info. Hope to see your contribution soon.

**Pull request closed without merging**